PolyclinicLab/Polyclinic.Api.Host/Program.cs

```diff
@@ -0,0 +1,73 @@
+﻿using Polyclinic.Domain.Models;
+using Polyclinic.Domain.Interfaces;
+using Polyclinic.Application.Services;
+using Polyclinic.Application.Interfaces;
+using Polyclinic.Application.MappingProfiles;
+using Polyclinic.Infrastructure.Mongo.Repositories;
+using Polyclinic.Infrastructure.Mongo.Context;
+using Polyclinic.Infrastructure.Mongo.Migrations;
+using Polyclinic.ServiceDefaults;
+using MongoDB.Driver;
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.AddServiceDefaults();
+builder.Services.AddControllers();
+builder.Services.AddEndpointsApiExplorer();
+builder.Services.AddSwaggerGen();
+builder.Services.AddAutoMapper(typeof(AppointmentMappingProfile));
+
+// Register MongoDB client (Aspire will provide connection string via ConnectionStrings:mongodb)
+var mongoConnectionString = builder.Configuration.GetConnectionString("mongodb");
+var mongoClient = new MongoClient(mongoConnectionString);
+builder.Services.AddSingleton<IMongoClient>(mongoClient);
```

Comment:
Когда вы установите пакет для работы с Aspire эти строки кода уйдут и сведутся к одной строчке.

```suggestion
// Register MongoDB client (Aspire will provide connection string via ConnectionStrings:mongodb)
builder.AddMongoDBClient(connectionName: "polyclinic");
```